src/bin/add/args.rs

```diff
@@ -202,10 +209,12 @@ impl Args {
                             prefix = self.get_upgrade_prefix(),
                             version = package.version
                         );
+
                         dependency = dependency.set_version(&v);
-                        if let Some(registry) = &self.registry {
-                            dependency = dependency.set_registry(registry);
-                        }
+                    }
+
+                    if let Some(registry) = &self.registry {
+                        dependency = dependency.set_registry(registry);
                     }
```

Comment:
Why is `set_registry` no longer in the block?

Reply:
mistake on my part. fixed it.